index.html

```diff
@@ -289,6 +289,20 @@ <h2>About me</h2>
           </ul>
         </div>
       </div>
+      <div class="contact-form">
+        <div class="description-text">
+          I'm always interested in hearing about new projects, 
+          so if you'd like to chat please get in touch.
+        </div>
+        <div class="form">
+          <form>
+            <input type="text" required maxlength="30" placeholder="Full name">
+            <input type="email" required placeholder="Email address">
+            <textarea maxlength="400" placeholder="Enter text here"></textarea>
```

Comment:
- [ ] Although the previous reviewer missed to point this out which is key in your portfolio website and also the building block in your project requirement(s)
> - Your form should have `action` and `method` attributes that foster integration with [formspree service](https://formspree.io/html) to help you get notified when a user makes a submission to your form. 
Please, kindly fix that also.

> - As required by the previous reviewer, kindly update your `textarea` to accept 500 characters which is also part of your project requirement(s)